gnocchi/incoming/_carbonara.py

```diff
@@ -75,6 +75,15 @@ def get_sack_lock(coord, sack):
     _SERIALIZE_DTYPE = [('timestamps', '<datetime64[ns]'),
                         ('values', '<d')]
 
+    def _make_measures_array(self):
```

Comment:
I'm feeling we should have a class that represent measures. Instead of all these almost static methods in the middle of the storage logic to manipulate the mesures array.